src/main/java/edu/cornell/kfs/module/cg/document/validation/impl/AwardExtensionRule.java

```diff
@@ -91,6 +92,29 @@ public void setupBaseConvenienceObjects(MaintenanceDocument document) {
         super.setNewBo(newAwardCopy);
     }
 	
+    /**
+     * checks if the required federal pass through agency number is filled in if the
+     * federal pass through indicator is yes
+     *
+     * @return success
+     */
+    @Override
+    protected boolean checkFederalPassThrough() {
+        boolean success = true;
+
+        boolean federalPassThroughIndicator = newAwardCopy.getFederalPassThroughIndicator();
+        String federalPassThroughAgencyNumber = newAwardCopy.getFederalPassThroughAgencyNumber();
+
+        if (federalPassThroughIndicator && StringUtils.isBlank(federalPassThroughAgencyNumber)) {
+            // fpt agency number is required if fpt indicator is checked

```

Comment:
I would recommend removing this comment, since the Javadoc comments already describe what this method is doing.